demo/conditional_demo.py

```diff
@@ -0,0 +1,124 @@
+import argparse
+import os
+import sys
+
+import mmcv
+from mmcv import DictAction
+from torchvision import utils
+
+# yapf: disable
+sys.path.append(os.path.abspath(os.path.join(__file__, '../..')))  # isort:skip  # noqa
+
+from mmgen.apis import init_model, sample_conditional_model  # isort:skip  # noqa
+# yapf: enable
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Generation demo')
+    parser.add_argument('config', help='test config file path')
+    parser.add_argument('checkpoint', help='checkpoint file')
+    parser.add_argument(
+        '--save-path',
+        type=str,
+        default='./work_dirs/demos/conditional_samples.png',
+        help='path to save uncoditional samples')
+    parser.add_argument(
+        '--device', type=str, default='cuda:0', help='CUDA device id')
+
+    # args for inference/sampling
+    parser.add_argument(
+        '--num-batches', type=int, default=4, help='Batch size in inference')
+    parser.add_argument(
+        '--num-samples',
+        type=int,
+        default=12,
+        help='The total number of samples')
```

Comment:
You should specify the usage of this argument. This argument will be used only if `not sample-all-class`